sql/core/src/test/scala/org/apache/spark/sql/expressions/ExpressionInfoSuite.scala

```diff
@@ -105,11 +105,28 @@ class ExpressionInfoSuite extends SparkFunSuite with SharedSparkSession {
     }
   }
 
+  test("SPARK-32870: Default expressions in FunctionRegistry should have their " +
+    "usage, examples and since filled") {
+    val ignoreSet = Set(
+      "org.apache.spark.sql.catalyst.expressions.TimeWindow")
```

Comment:
Why did you put `TimeWindow` in this list?

Reply:
Based on this discussion https://github.com/apache/spark/pull/29743#discussion_r487594286

Reply:
Ah, I see. Please leave comments here about it?